packages/@react-spectrum/color/src/ColorWheel.tsx

```diff
@@ -92,7 +92,7 @@ function ColorWheel(props: SpectrumColorWheelProps, ref: FocusableRef<HTMLDivEle
       }}>
       <div {...trackProps} className={classNames(styles, 'spectrum-ColorWheel-gradient')} />
       <ColorThumb
-        value={state.value}
+        value={parseColor(`hsl(${state.value.getChannelValue('hue')}, 100%, 50%)`)}
```

Comment:
what was happening here?

Reply:
since the visual colors represented by the ColorWheel are of a fixed saturation and lightness [value](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/color/src/useColorWheel.ts#L244-L258), it felt odd that the thumb would actually accept the user defined saturation and lightness values.

A ColorWheel that had value of `hsl(0, 56%, 23%)` would have a thumb of the following color (aka it mirrors the user's saturation and lightness values rather than keeping the color wheel's static saturation and lightness values):
![image](https://user-images.githubusercontent.com/9661127/111012087-83fdc200-8350-11eb-953c-c5f28ffc624e.png)

I'm fine with reverting this change, just felt like a odd user experience to see a non matching color between the thumb and the wheel itself

Reply:
Ok makes sense but having it parse in the render is not very efficient. 
`color.withChannelValue('saturation', 100).withChannelValue(50)` ?

Add getDisplayColor to stately like ColorSlider has? https://github.com/adobe/react-spectrum/blob/main/packages/%40react-stately/color/src/useColorSliderState.ts#L75

Reply:
Sounds good, I'll replace with `state.value.withChannelValue('saturation', 100).withChannelValue('lightness', 50)`. I don't know if adding getDisplayColor to `useColorWheel` is appropriate though, color wheels don't have a variety of channels to represent right? Usually they are only used to pick a hue.

Reply:
this seems like a good thing to have even if you aren't using Spectrum so having it in stately would be helpful.

Reply:
hm, but doesn't value give that? Or are we just trying to return hue + 100% saturation +50% lightness?

Reply:
> Or are we just trying to return hue + 100% saturation +50% lightness?

yes